packages/DefaultAutowire/src/DependencyInjection/DefinitionAnalyzer.php

```diff
@@ -138,14 +137,18 @@ private function haveMissingArgumentsTypehints(
                     continue;
```

Comment:
I think there should be `return false;` here too. You can't autowire if there is no typehint.